packages/client/hmi-client/src/components/models/forecat-launcher.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<Modal class="modal" @modal-mask-clicked="close()">
+		<template #default>
+			<!-- initial values -->
+			<h4>Initial values</h4>
+			<section>
+				<div v-for="(s, i) of props.model.content.S" :key="i" class="row">
+					<span>{{ s.sname }}</span>
+					<InputText type="text" class="p-inputtext-sm" v-model.number="initialValues[s.sname]" />
+				</div>
+			</section>
+
+			<!-- params -->
+			<h4>Parameter values</h4>
+			<section>
+				<div v-for="(t, i) of props.model.content.T" :key="i" class="row">
+					<span>{{ t.tname }}</span>
+					<InputText type="text" class="p-inputtext-sm" v-model.number="parameterValues[t.tname]" />
+				</div>
+			</section>
+		</template>
+		<template #footer>
+			<Button @click="launch">Launch</Button>
+			<Button class="p-button-secondary" @click="close()">Cancel</Button>
+		</template>
+	</Modal>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import { ITypedModel } from '@/types/Model';
+import { PetriNet } from '@/petrinet/petrinet-service';
+import Modal from '@/components/widgets/Modal.vue';
+import InputText from 'primevue/inputtext';
+import Button from 'primevue/button';
+import { makeForecast } from '@/services/models/simulation-service';
+
+interface NumericValueMap {
+	[key: string]: number;
+}
+
+const props = defineProps<{
+	model: ITypedModel<PetriNet>;
+}>();
+
+const initialValues = ref<NumericValueMap>({});
+const parameterValues = ref<NumericValueMap>({});
+
+props.model.content.S.forEach((s) => {
+	initialValues.value[s.sname] = 1;
+});
+props.model.content.T.forEach((s) => {
+	parameterValues.value[s.tname] = 0.5;
+});
+
+const emit = defineEmits(['close', 'launch-forecast']);
+
+const launch = async () => {
+	// FIXME: current need to strip out metadata
+	const cleanedModel: PetriNet = {
+		S: props.model.content.S.map((s) => ({ sname: s.sname })),
+		T: props.model.content.T.map((t) => ({ tname: t.tname })),
+		I: props.model.content.I,
+		O: props.model.content.O
+	};
+
+	const payload = {
+		petri: JSON.stringify(cleanedModel),
+		initials: initialValues.value,
+		params: parameterValues.value,
+		tspan: [0, 50]
+	};
+
+	const run = await makeForecast(payload);
+
+	// FIXME: Cache into sessionStorage
```

Comment:
You are right... we need to have a user management/database to handle this properly now.